locale/ru.po

```diff
@@ -153,7 +153,7 @@ msgstr "Продолжить установку? [Y/n]"
 
 #: pikaur/package_update.py:45
 msgid "Reading AUR packages info..."
-msgstr "Чтение информации пакетов из AUR..."
+msgstr "Чтение информации о пакетах из AUR..."
```

Comment:
I think we should have support for plurals here (in src) :)

Reply:
@Yardanico And I did it removed plural?

Reply:
@Mauladen I've meant that we maybe need add support for one/many packages in the actual pikaur source code for this line

Reply:
551c9f4ddf0a95e9a68709ed42ac122dfe3cf586

Reply:
@actionless I ask you to accept these changes, and will finalize the rest of @Yardanico